src/integration/pushResult.js

```diff
@@ -48,11 +52,39 @@ function clone(repo, branch, dest) {
 }
 
 //delete previous cloned results
-function deleteNotNeeded(independent, notUsedFiles){
+function backupOfNotClonedRepositories(dest, independent){
+  return (cb) => {
+    if (independent) return cb();
+
+    _backup(true, false, cb);
+  };
+}
+
+//delete previous cloned results
+function deleteNotNeeded(notUsedFiles, independent){
   return (cb) => {
     if (!independent) return cb();
 
-    del(notUsedFiles).then(cb);
+    del.sync(notUsedFiles);
+    cb();
+  };
+}
+
+//delete previous cloned results
+function prepareForCopy(dest, independent) {
```

Comment:
(just a suggestion) Maybe better : deletePreviousClonedResults, delPrevResults, delPrevRepo.. i mean prepareForCopy its not consistent with others :P
  clone(repo, branch, dest),
    backupOfNotClonedRepositories(dest, independent),
    deleteNotNeeded(notUsedFiles, independent),
    prepareForCopy(dest, independent),